queries.sql

```diff
@@ -22,6 +22,61 @@ select * from animals where name != 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 select * from animals where weight_kg between 10.4 and 17.3;
 
+-- Set the species column to unspecified inside a transaction then roll back
+begin;
+update animals
+set species = 'unspecified' where species is null;
+rollback;
 
+-- start a transaction 1
+begin;
+-- Update the animals table by setting the species column to 'digimon' for all animals that have a name ending in 'mon'
+update animals
+set species = 'digimon' where name like '%mon';
 
+-- Update the animals table by setting the species column to 'pokemon' for all animals that don't have species already set.
+update animals
+set species = 'pokemon' where species is null;
+-- Commit the trabsaction
+commit;
 
+-- start a transaction 2
+begin;
+-- delete all records in the animals
+delete from animals;
+-- roll back the transaction
+rollback;
```

Comment:
Hello, I'm here to review your code. You have done a partial job on this project.

One thing I noticed is that you did not include any SELECT statements to verify that your change was made and rolled back. The project requirements explicitly ask for this step, so you should add two SELECT statements after your DELETE and ROLLBACK statements, respectively. You should also check the output of your SELECT statements to make sure that they match the expected results.

Here's an example of how you can write your code that meets all the project requirements:

```sql
BEGIN; -- start transaction

-- Delete all records in the animals table
DELETE FROM animals;

-- Verify that change was made
SELECT COUNT(*) FROM animals;

-- Roll back the transaction
ROLLBACK;

-- Verify that change was rolled back
SELECT COUNT(*) FROM animals;

Reply:
Hi @topeogunleye , I really appreciate the time and effort you invested in reviewing my project.  The `select` statements are used in the second screenshot in the PR description. Anyway, I also added `select count` statement where required for the third transaction.

Thank you for your feedback.

Reply:
Hi, @momo-87,

You’re very welcome. I’m glad to hear that you made the necessary changes to your project. You have done a great job of following the instructions and applying the SQL concepts. Your code is well-written, clear, and efficient.